### DISCUSSION TITLE
How to copy visual selection to system clipboard?

### DISCUSSION THREAD

**Discussion by dynobo** [vscode-neovim/vscode-neovim#1636] [Q&A] [answered]
What exactly do I have to do to be able to copy a selection in visual mode to the system clipboard? It would be awesome, if someone could explain it to me step by step.

I have read  the question "[how to yank from VSCode neovim to system clipboard?](https://github.com/vscode-neovim/vscode-neovim/discussions/1179)" along it's [corresponding issue](https://github.com/vscode-neovim/vscode-neovim/issues/835#issuecomment-1073498454) and the issue "[Use <C-c> to copy visually selected text to clipboard](https://github.com/vscode-neovim/vscode-neovim/issues/1196)", but honestly, I'm still lost: Both approaches seem quite different to me, and both approaches where recommended before #1258 got merged, which also seems related to this feature somehow? Appreciate any help.

PS: Thanks a lot for this useful extension! :heart:

**Comment by justinmk** [comment, marked as answer]
```
v
...select some text...
"+y
```

**Reply by dynobo** [reply]
Now I feel a little bit stupid... :see_no_evil: 
Thanks! :slightly_smiling_face:
